bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/dependencies/sqlite/listings/ListingsTableAccessor.java

```diff
@@ -63,15 +63,49 @@ public List<Listing> getAllListings() throws SQLException {
 		return listings;
 	}
 
-	public Listing getListingById(int listingId) {
-		// TODO
+	public Listing getListingById(int listingId) throws SQLException {
+		String sql = "SELECT * FROM listings WHERE listing_Id = ?",
+
+		Connection connection = getDatabaseConnection();
+		PreparedStatement statement = connection.prepareStatement(sql);
+
+		statement.setInt(1, listingId);
+
+		try(ResultSet resultSet = statement.executeQuery()) {
+			if(resultSet.next()) {
+				String author = results.getString("author");
+				String title = results.getString("title");
+				float price = results.getFloat("price");
+				String description = results.getString("description");
+				int sellerId = results.getInt("seller_id");
+				String classSubject = results.getString("class_subject");
+				String condition = results.getString("condition");
+				String saleAvailability = results.getString("sale_availability");
+				Instant createdAt = Instant.parse(results.getString("created_at"));
+
+				return new Listing(
+						listingId,
+						author,
+						title,
+						price,
+						description,
+						sellerId,
+						classSubject,
+						condition,
+						saleAvailability,
+						createdAt,
```

Comment:
Adding a comma after the last supplied argument of a constructor is a syntax error, we should remove this before merging